starlite/testing/test_client/client.py

```diff
@@ -653,42 +676,35 @@ def test_something(self, test_client: TestClient) -> None:
         return self.get_session_data()
 
     @staticmethod
-    def _create_session_cookies(backend: CookieBackend, data: Dict[str, Any]) -> Dict[str, str]:
+    def _create_session_cookies(backend: "CookieBackend", data: Dict[str, Any]) -> Dict[str, str]:
         encoded_data = backend.dump_data(data=data)
         return {cookie.key: cast("str", cookie.value) for cookie in backend._create_session_cookies(encoded_data)}
 
     async def _set_session_data_async(self, data: Dict[str, Any]) -> None:
         # TODO: Expose this in the async client
-        if isinstance(self.session_backend, ServerSideBackend):
-            serialized_data = self.session_backend.serlialize_data(data)
-            session_id = self.cookies.setdefault(
-                self.session_backend.config.key, self.session_backend.generate_session_id()
-            )
-            await self.session_backend.set(session_id, serialized_data)
-        elif isinstance(self.session_backend, CookieBackend):
-            for key, value in self._create_session_cookies(self.session_backend, data).items():
-                self.cookies.set(key, value)
-        else:
-            raise_for_unsupported_session_backend(self.session_backend)
+        mutable_headers = MutableHeaders({})
```

Comment:
You can simply use a dict here i think

Reply:
I could, but then I'd have to convert the dict to `MutableHeaders` (or `Headers`) in `fake_http_send_message` (for converting them to a bytes list), and IMO it's a bit clearer like this, since the same object gets used throughout.